downstairs/src/lib.rs

```diff
@@ -2749,7 +2738,7 @@ impl Downstairs {
 
     async fn run(
```

Comment:
We should add some comments here to signify this is now the "main loop" for
doing work on the downstairs.

Reply:
Done in 2c93629b2210